src/NUnitTestAdapterTests/TestDiscoveryTests.cs

```diff
@@ -65,7 +65,7 @@ public TestDiscoveryTests(IDiscoveryContext context)
         public void LoadMockassembly()
         {
             // Sanity check to be sure we have the correct version of mock-assembly.dll
-            Assert.That(NUnit.Tests.Assemblies.MockAssembly.Tests, Is.EqualTo(34),
+            Assert.That(NUnit.Tests.Assemblies.MockAssembly.TestsAtRuntime, Is.EqualTo(NUnit.Tests.Assemblies.MockAssembly.Tests),
```

Comment:
I understand why this works, but the hard-coded number gives me a lot more confidence. That said, we may not need the check at all. it was inherited from the framework project, which had several different versions of mock-assembly.